js/base-jsload.js

```diff
@@ -38,11 +38,13 @@ function baseJsLoad () {
 
 		// Monkey patch JSON loading
 		const cached = DataUtil.loadJSON;
-		DataUtil.loadJSON = async (url, ...others) => {
+		DataUtil.loadJSON = async (url, forceExternal=false, ...others) => {
```

Comment:
prefer `{isForceExternal = false} = {}` here, so we can add other options in future, and because prefixing definitely-boolean variables with `is` or `has` is nice